backend/src/controllers/recipe.ts

```diff
@@ -13,4 +14,15 @@ export default {
       next(error);
     }
   },
+  search: async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      checkQueryParams(['title', 'description'], req.query);
+      const { title, description } = req.query as { title?: string; description?: string; };
```

Comment:
```suggestion
      const { title, description } = req.query as { title: string; description: string; };
```

Comme tu viens de check le type au dessus